src/ui/server/shared-validation/whole-number-minimum-maximum-length/index.ts

```diff
@@ -0,0 +1,51 @@
+import { RequestBody, ErrorMessageObject } from '../../../types';
+import { objectHasProperty } from '../../helpers/object';
+import { stripCommas } from '../../helpers/string';
+import generateValidationErrors from '../../helpers/validation';
+import wholeNumberValidation from '../../helpers/whole-number-validation';
+
+/**
+ * wholeNumberMinimumMaximumLength
+ * Check if a number is provided and its length is within the minimum and maximum length
+ * @param {RequestBody} formBody
+ * @param {String} fieldId
+ * @param {String} errorMessage
+ * @param {Object} errors object from previous validation errors
+ * @param {Number} minimum minimum length to check against
+ * @param {Number} maximum maximum length to check against
+ * @returns {Object} Validation errors or null
+ */
+const wholeNumberMinimumMaximumLength = (
+  formBody: RequestBody,
+  fieldId: string,
+  errorMessage: ErrorMessageObject,
+  errors: object,
+  minimum?: number,
+  maximum?: number,
+) => {
+  let updatedErrors = errors;
+
+  if (!objectHasProperty(formBody, fieldId)) {
+    return generateValidationErrors(fieldId, errorMessage.IS_EMPTY, errors);
+  }
+
+  // strip commas - commas are valid.
+  const numberWithoutCommas = stripCommas(formBody[fieldId]);
+
+  // check if the field is a whole number.
+  updatedErrors = wholeNumberValidation(formBody, updatedErrors, errorMessage.INCORRECT_FORMAT, fieldId);
+
+  // check if the field is below the minimum length
+  if (minimum && numberWithoutCommas.length < minimum) {
+    return generateValidationErrors(fieldId, errorMessage.BELOW_MINIMUM, errors);
+  }
+
+  // check if the field is above the maximum length
+  if (maximum && numberWithoutCommas.length > maximum) {
+    return generateValidationErrors(fieldId, errorMessage.ABOVE_MAXIMUM, errors);
+  }
+
```

Comment:
Same as above, maybe can use `isNumberBelowMinimum` and `isNumberAboveMaximum`?

Should the minimum and maximum variables here be required? I can't see why they'd be optional